extensions/3DTILES_bounding_volume_S2/README.md

```diff
@@ -0,0 +1,459 @@
+- Peter Gagliardi, Cesium
+
+<!-- omit in toc -->
+## Status
+
+Draft
+
+<!-- omit in toc -->
+## Dependencies
+
+Written against 3D Tiles 1.0. Optionally, this extension may be used with [3DTILES_implicit_tiling](../3DTILES_implicit_tiling). 
+
+<!-- omit in toc -->
+## Optional vs. Required
+
+This extension is required, meaning it must be placed in both the `extensionsUsed` and `extensionsRequired` lists in the tileset JSON.
+
+<!-- omit in toc -->
+## Contents
+
+- [Overview](#overview)
+- [Hierarchy](#hierarchy)
+- [Cell IDs](#cell-ids)
+- [Tokens](#tokens)
+- [Bounding Volume](#bounding-volume)
+- [Implicit Subdivision](#implicit-subdivision)
+  - [Availability](#availability)
+- [Property Reference](#property-reference)
+- [JSON Schema](#json-schema)
+- [Implementation Examples](#implementation-examples)
+
+## Overview
+
+S2 is a geometry library that defines a framework for decomposing the unit sphere into a hierarchy of cells. A cell is a quadrilateral bounded by four geodesics. The S2 cell hierarchy has 6 root cells, that are obtained by projecting the six faces of a cube onto the unit sphere.
+
+Typically, traditional GIS libraries rely on projecting map data from an ellipsoid onto a plane. For example, the Mercator projection is a cylindrical projection, where the ellipsoid is mapped onto a cylinder that is then unrolled as a plane. This method leads to increasingly large distortion in areas as you move further away from the equator. S2 makes it possible to split the Earth into tiles that have no singularities, have relatively low distortion and are nearly equal in size.
+
+This extension to 3D Tiles enables using S2 cells as bounding volumes. Due to the properties of S2 described above, this extension is well suited for tilesets that span the whole globe.
+
+## Hierarchy
+
+The cell hierarchy of S2 is rooted in the 6 faces of a cube, which are projected onto the unit sphere. In S2, each face of the unit cube can be subdivided into 30 levels using a quadtree structure, in which each cell on the grid subdivides into 4 equal cells at the subsequent level.
+
+|Level 0|Level 1|
+|:-:|:-:|
+| ![Plane - Level 0](figures/plane_parent.png) S2 cell (`"1"`) on the Earth cube| ![Plane - Level 1](figures/plane_children.png) Children of S2 cell (`"1"`) on the Earth cube |
+| ![Ellipsoid - Level 0](figures/ellipsoid_parent.png)  S2 cell (`"1"`) on the WGS84 ellipsoid| ![Ellipsoid - Level 1](figures/ellipsoid_children.png) Children of S2 cell (`"1"`) on the WGS84 ellipsoid|
+
+The S2 library uses a modified Hilbert curve to provide a one dimensional ordering of cells on the S2 Earth cube. This provides each cell, from level 1 to level 30, with a unique 64-bit identifier. Using S2 cell IDs, centimeter scale areas be uniquely identified.
+
+| S2 Curve on Earth cube |  S2 Curve on WGS84 ellipsoid |
+|:-:|:-:|
+| ![Math](figures/plane.png)  | ![Math](figures/ellipsoid.png)  |
+
+## Cell IDs
+
+The 64-bit S2 cell ID is constructed as follows:
```

Comment:
@sanjeetsuhag is there something I can link to where S2 cell IDs are officially defined?

Reply:
The "official" explainer would be this: https://s2geometry.io/devguide/s2cell_hierarchy